.github/workflows/lint_code.yml

```diff
@@ -17,6 +17,4 @@ jobs:
       run: pip install flake8
```

Comment:
I think on master, `pip install flake8` is actually pinned to `<6.0.0`? So when we merge master, we'll probably have to add another commit to unpin `flake8`.

Reply:
Good point. Although that could even be a separate PR if `flake8` version 6.0.0 introduces other breaking changes.